### ISSUE TITLE
Integrated Terminal or a way to Run my code without opening up my command prompt

### ISSUE THREAD

**Issue by mrinal1894** [atom/atom#21810]
## Summary

Can you please add an integrated terminal or maybe a run button, so that executing a code in some directory can be done without opening command prompt ?

## Motivation

I have been using atom for a year now. I mostly code in python and enjoy coding in atom. Despite being a text editor, I personally believe that atom has a good interface when it comes to interacting with github without installing any third party packages. This speeds up my work however when it comes to executing my code, I have to open my command prompt and run the file from there. This is a problem for projects having multiple folders as to run a file present in some other folder, I have to change my directory in atom and then change it again in my command prompt.

What I expect is that to run some file present in some folder, changing the directory once in atom should be enough and then to run the file, either I run it directly from the atom terminal or there could be a run button which I could press to execute it and display the result in some window.

## Describe alternatives you've considered

There is a Third party package "platformio-ide-terminal" which is the most popular terminal package available on atom however it is extremely buggy. It's very unstable to use and only works for some time. Same things are true for other terminal packages available.  

I am attaching a screenshot of the platformio-ide-terminal. It's not opening. Only a blank screen appears. I read online about some possible fixes to this problem but none seems to be working. That's why I don't want to rely on any third party package.

![Screenshot (2)](https://user-images.githubusercontent.com/52851566/102657484-c2de0a80-419b-11eb-9cad-00711a43a593.png)

**Comment by UziTech** [comment]
platformio-ide-terminal is no longer maintained. That is why it is so buggy. You can try [terminus](https://atom.io/packages/terminus) or [x-terminal](https://atom.io/packages/x-terminal).

There is an issue #14490 that is about adding a terminal directly to Atom.

TL;DR A terminal that is built into Atom wouldn't be any better (or less buggy) than a third party terminal that is well maintained.

**Comment by ThatXliner** [comment]
I saw an "atom terminal" in the atom archive. They used to have it built-in.

**Comment by UziTech** [comment]
They have a placeholder package ([terminal](https://atom.io/packages/terminal)). I think they had plans to add it at one point but I don't think they ever did.
